image_to_skill/__main__.py

```diff
@@ -7,46 +7,189 @@
+
+        self.parti_type_var = StringVar()
+        self.parti_type_entry = Entry(self.frame_inside_frame, width=15, font=('Arial', 11),
+                                      textvariable=self.parti_type_var)
+        self.parti_type_entry.grid(row=0, column=1, padx=10)
+
+        self.parti_interval_label = Label(self.frame_inside_frame, font=('Arial', 11), text='ParticleInterval:')
+        self.parti_interval_label.grid(row=0, column=2)
+
+        self.parti_interval_var = StringVar()
+        self.parti_interval_entry = Entry(self.frame_inside_frame, font=('Arial', 11), width=15,
+                                          textvariable=self.parti_interval_var)
+        self.parti_interval_entry.grid(row=0, column=3, padx=5)
+
+        self.parti_size_label = Label(self.frame_inside_frame, font=('Arial', 11), text='ParticleSize:')
+        self.parti_size_label.grid(row=1, column=0, pady=5)
+
+        self.parti_size_var = StringVar()
+        self.parti_size_entry = Entry(self.frame_inside_frame, font=('Arial', 11), width=15,
+                                      textvariable=self.parti_size_var)
+        self.parti_size_entry.grid(row=1, column=1, padx=10)
+
+        self.forward_offset_label = Label(self.frame_inside_frame, font=('Arial', 11), text='BaseForwardOffset:')
+        self.forward_offset_label.grid(row=1, column=2)
+
+        self.forward_offset_var = StringVar()
+        self.forward_offset_entry = Entry(self.frame_inside_frame, font=('Arial', 11), width=15,
+                                          textvariable=self.forward_offset_var)
+        self.forward_offset_entry.grid(row=1, column=3, pady=5)
+
+        self.side_offset_label = Label(self.frame_inside_frame, font=('Arial', 11), text='BaseSideOffset:')
+        self.side_offset_label.grid(row=2, column=0)
+
+        self.side_offset_var = StringVar()
+        self.side_offset_entry = Entry(self.frame_inside_frame, width=15, font=('Arial', 11),
+                                       textvariable=self.side_offset_var)
+        self.side_offset_entry.grid(row=2, column=1, padx=10)
+
+        self.y_offset_label = Label(self.frame_inside_frame, font=('Arial', 11), text="BaseYOffset:")
+        self.y_offset_label.grid(row=2, column=2)
+
+        self.y_offset_var = StringVar()
+        self.y_offset_entry = Entry(self.frame_inside_frame, font=('Arial', 11), width=15,
+                                    textvariable=self.y_offset_var)
+        self.y_offset_entry.grid(row=2, column=3, padx=5)
+
+        # 帮助按钮
+        self.help_btn = Button(self.frame_inside_frame, text='Help', width=8, takefocus=False, command=self.get_help)
+        self.help_btn.grid(row=3, column=1, pady=5)
+
+        self.particle_help = Button(self.frame_inside_frame, text='ParticleHelp', width=11, takefocus=False,
+                                    command=self.get_help_particle)
+        self.particle_help.grid(row=3, column=2)
+
+        self.root.mainloop()
```

Comment:
We need to now interface this with the back-end code.

Reply:
Whoops. Interfacing has been implemented in `main(self)` but accidentally skipped it. My bad.